enhancements/ingress/aws-load-balancer-operator.md

```diff
@@ -0,0 +1,394 @@
+---
+title: aws-load-balancer-operator 
+authors:
+  - "@arjunrn"
+reviewers:
+  - "@Miciah"
+  - "@alebedev87"
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2022-01-27
+last-updated: 2022-01-27
+tracking-link:
+  - https://issues.redhat.com/browse/CFEPLAN-39
+see-also:
+  - "/enhancements/ingress/transition-ingress-from-beta-to-stable.mde"
+replaces:
+superseded-by:
+---
+
+# AWS Load Balancer Operator
+
+## Summary
+
+Users would like to use [Application Load
+Balancers](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html)(ALBs)
+for their Ingress resources so that they can use [Amazon Certificate
+Manager](https://aws.amazon.com/certificate-manager/)(ACM) to manage
+certificates for the Ingress domains and also attach [Web Application
+Firewalls](https://aws.amazon.com/waf/) to the ALB for security and traffic
+shaping. The Openshift [router](github.com/openshift/router) uses an NLB which
+passes through all traffic to the router. TLS termination is done on the router.
+A WAF cannot be attached to the NLB and the router cannot be configured to
+perform TLS termination at the NLB.
```

Comment:
The ingress operator configures a classic ELB on AWS by default, with NLB as an option.  
```suggestion
Users would like to use [Application Load
Balancers](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html) (ALBs)
for their Ingress resources so that they can use [Amazon Certificate
Manager](https://aws.amazon.com/certificate-manager/) (ACM) to manage
certificates for the Ingress domains and also attach [Web Application
Firewalls](https://aws.amazon.com/waf/) (WAFs) to the ALB for security and traffic
shaping. The OpenShift [router](https://github.com/openshift/router) uses a classic ELB or an NLB, which
passes through all traffic to the router. TLS termination is done on the router or backend pod.
A WAF cannot be attached to the ELB or NLB, and the router cannot be configured to
perform TLS termination at the ELB or NLB.
```